pkg/volume/iscsi/attacher.go

```diff
@@ -81,6 +81,10 @@ func (attacher *iscsiAttacher) WaitForAttach(spec *volume.Spec, devicePath strin
 		klog.Warningf("failed to get iscsi mounter: %v", err)
 		return "", err
 	}
+	exists, err := mount.PathExists(devicePath)
+	if !exists && err != nil {
```

Comment:
It does not make sense to check `devicePath`, it's always empty (as returned by `Attach()` above)

Reply:
@jsafrane should I close this PR.
Also could you please review the PR for fc disk, https://github.com/kubernetes/kubernetes/pull/107279#

Reply:
apply the same change with #107279 ?
```suggestion
	if exists && err == nil {
		return devicePath, nil
	}
	if exists == false {
		return "", fmt.Errorf("device %s does not exist", devicePath)
	} else {
		return "", err
```

Reply:
Again, `devicePath` is always empty. It really does not make any sense to check it here.